step-templates/roundhouse-database-migration.json

```diff
@@ -0,0 +1,257 @@
+        }
+      },
+      {
+        "Id": "9ddd49a7-92a9-419a-aa07-70c569d88540",
+        "Name": "roundhouseDatabaseServerType",
+        "Label": "Database Server Type",
+        "HelpText": "The database technology being deployed to.",
+        "DefaultValue": "",
+        "DisplaySettings": {
+          "Octopus.ControlType": "Select",
+          "Octopus.SelectOptions": "mysql|MariaDB\nmysql|MySQL\noracle|Oracle\npostgres|PostgreSQL\nsqlserver|SQL Server"
+        }
+      },
+      {
+        "Id": "0d748383-fd88-4dc2-ad8e-7b6f12137e3b",
+        "Name": "roundhouseWithTransaction",
+        "Label": "Use Transaction?",
+        "HelpText": "Check this box if you want all scripts to be run within the same transaction.",
+        "DefaultValue": "False",
+        "DisplaySettings": {
+          "Octopus.ControlType": "Checkbox"
+        }
+      },
+      {
+        "Id": "06f538af-087a-4d59-958d-90c2e4decebf",
+        "Name": "roundhouseDryRun",
+        "Label": "Dry Run?",
+        "HelpText": "Check this box if you want to perform a dry run.  Results are recorded and attached as deployment artifacts if you check Record Output.",
+        "DefaultValue": "False",
+        "DisplaySettings": {
+          "Octopus.ControlType": "Checkbox"
+        }
+      },
+      {
+        "Id": "ec772c86-c12a-4f4f-8710-6426152faa72",
+        "Name": "roundhouseRecordOutput",
+        "Label": "Record Output?",
+        "HelpText": "Check this box to record the output of the run.  Useful for gathering what would be changed for approval purposes.",
+        "DefaultValue": "False",
+        "DisplaySettings": {
+          "Octopus.ControlType": "Checkbox"
+        }
+      },
+      {
+        "Id": "46ee4671-67b2-4e45-b484-6773b41c2bf8",
+        "Name": "roundhouseUpFolder",
+        "Label": "Up Folder Path",
+        "HelpText": "Relative location of the Up folder.  If left blank, .\\up is used.",
+        "DefaultValue": "",
+        "DisplaySettings": {
+          "Octopus.ControlType": "SingleLineText"
+        }
+      },
+      {
+        "Id": "55e0bf86-b675-4204-9e0b-d39d20b15eea",
+        "Name": "roundhouseAlterDatabaseFolder",
+        "Label": "AlterDatabase Folder Path",
+        "HelpText": "AlterDatabase folder path.  If left blank, .\\alterDatabase is used.",
+        "DefaultValue": "",
+        "DisplaySettings": {
+          "Octopus.ControlType": "SingleLineText"
+        }
+      },
+      {
+        "Id": "cd473440-f9a9-4a5d-b48c-46ee63a92e7a",
+        "Name": "roundhouseRunBeforeUpFolder",
+        "Label": "RunBeforeUp Folder Path",
+        "HelpText": "RunBeforeUp folder path.  If left blank, .\\runbeforeUp is used.",
+        "DefaultValue": "",
+        "DisplaySettings": {
+          "Octopus.ControlType": "SingleLineText"
+        }
+      },
+      {
+        "Id": "53fab9f0-f310-482c-985c-e6efdf206117",
+        "Name": "roundhouseFunctionsFolder",
+        "Label": "FunctionsFolder Path",
```

Comment:
This param and the ones after it have Folder concatenated to the folder name. The ones before it have them separate. Can we make this one and the following separate words?

Reply:
Done.